lib/internal/modules/esm/worker.js

```diff
@@ -74,7 +74,9 @@ function wrapMessage(status, body) {
 }
 
 async function customizedModuleWorker(lock, syncCommPort, errorHandler) {
-  let hooks, preloadScripts, initializationError;
+  let hooks;
+  let preloadScripts;
+  let initializationError;
```

Comment:
Cosmetic change, please remove

Reply:
As I’ve explained, it’s not cosmetic. This reduces the potential for backport conflicts, which is the entire goal of this PR.